app/views/articles/edit.html.erb

```diff
@@ -1,5 +1 @@
-<h1 class="text-3xl text-center font-bold">Edit Article</h1>
-
-<br>
-
-<%= render 'form', model: @article %>
\ No newline at end of file
+<%= render partial: 'form', locals: { model: @article, mode: 'Edit' } %>
```

Comment:
@J-pilon your previous comment got removed because I pushed changes, but yes you can totally just use `'form'` when using the partials! Thank you for the tip 😄